src/pages/junghoon/Login/Login.js

```diff
@@ -1,13 +1,108 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import "./Login.scss";
+import "../../../styles/common.scss";
 
 class Login extends Component {
+  constructor(e) {
+    super(e);
+
+    this.state = {
+      userId: "",
+      userPw: "",
+    };
+  }
+
+  // goToMain = () => {
+  //   this.props.history.push("/mainPark");
+  // };
+
+  handleIdInput = (e) => {
+    this.setState({
+      userId: e.target.value,
+    });
+  };
+
+  handlePwInput = (e) => {
+    this.setState({
+      userPw: e.target.value,
+    });
+  };
+
+  onClick = () => {
+    fetch("http://10.58.6.173:8000/user/signin", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.userId,
+        password: this.state.userPw,
+        name: "정훈1121",
+        phone: "01048780454",
+        nickname: "나1나나112",
+      }),
+    })
+      .then((response) => response.json())
+      .then((result) => {
+        console.log(result);
+        if (result.message === "SUCCESS") {
+          this.props.history.push("/mainPark");
+          localStorage.setItem("TOKEN", result.token);
```

Comment:
```suggestion
          localStorage.setItem("TOKEN", result.token);
          this.props.history.push("/mainPark");
```
토큰 값을 저장한 후에 페이지 이동을 해주세요~